### PR TITLE
Add --tags and --skip-tags options

### PR DESCRIPTION
**Pull request by seleznev**
Иллюстрация к #80.

### CODE REVIEW

File: k8s_handle/templating.py [hunk 1]
```diff
@@ -113,6 +116,40 @@ def generate_by_context(self, context):
...
+        return [i for i in templates if self._evaluate_tags(i, only_tags, skip_tags)]
+
+    def _evaluate_tags(self, template, only_tags, skip_tags):
+        if 'tags' in template:
+            tags = template['tags']
+
+            if isinstance(tags, list):
+                tags = set([i for i, _ in itertools.groupby(tags)])
+            elif isinstance(tags, str):
+                if tags.find(',') != -1:
```
Comment (rvadim) [inline k8s_handle/templating.py]: Condition not required:
```
In [1]: 'hello'.split(',')                                                                                              
Out[1]: ['hello']
```

File: k8s_handle/templating.py [hunk 2]
```diff
@@ -113,6 +116,40 @@ def generate_by_context(self, context):
                 raise TemplateRenderingError('Unable to render {}, due to: {}'.format(template, e))
         return output
 
+    def _filter_tagged_templates(self, templates, only_tags, skip_tags):
+        return [i for i in templates if self._evaluate_tags(i, only_tags, skip_tags)]
```
Comment (rvadim) [inline k8s_handle/templating.py]: More idiomatic way:
```
return templates.filter(lambda i: self._evaluate_tags(i, only_tags, skip_tags)) 
```

Comment (rvadim) [reply k8s_handle/templating.py]: So, maybe `def _filter_tagged_templates(self, templates, only_tags, skip_tags)`, not required at all.

File: k8s_handle/templating.py [hunk 3]
```diff
@@ -113,6 +116,40 @@ def generate_by_context(self, context):
...
+
+    def _evaluate_tags(self, template, only_tags, skip_tags):
```
Comment (rvadim) [inline k8s_handle/templating.py]: The method return `true` or `false`, maybe `_is_template_tagged(...)` more accurate name?

File: k8s_handle/templating.py [hunk 4]
```diff
@@ -113,6 +116,40 @@ def generate_by_context(self, context):
...
+        if 'tags' in template:
```
Comment (rvadim) [inline k8s_handle/templating.py]: I think we need to separate template tags analysis and skipping logic to 2 different functions.

**Comment by rvadim** [comment]
Aslo, don't forget to add description about this feature in README.md

File: k8s_handle/__init__.py [hunk 5]
```diff
@@ -86,6 +90,8 @@ def main():
     settings.CHECK_STATUS_TIMEOUT = args.get('retry_delay')
     settings.CHECK_DAEMONSET_STATUS_TIMEOUT = args.get('retry_delay')
     settings.GET_ENVIRON_STRICT = args.get('strict')
+    settings.ONLY_TAGS = args['tags']
```
[the remark this answers is not in the document]
Comment (seleznev) [reply k8s_handle/__init__.py]: Oooops. Fixed.

File: k8s_handle/templating.py (hunk 1 above)
Comment (seleznev) [reply k8s_handle/templating.py]: Fixed.

File: k8s_handle/templating.py (hunk 4 above)
Comment (seleznev) [reply k8s_handle/templating.py]: I moved first part into `_get_template_tags()`.

File: k8s_handle/templating.py (hunk 3 above)
Comment (seleznev) [reply k8s_handle/templating.py]: I don't like both names actually (especially after moving "template" part into separate function). :slightly_frowning_face:

Please let me know if `_is_template_tagged(self, tags, only_tags, skip_tags)` still looks better for you after last changes.

File: k8s_handle/templating.py (hunk 2 above)
Comment (seleznev) [reply k8s_handle/templating.py]: Ok, I replaced `_filter_tagged_templates()` to `filter()` + `lambda`:
```python
templates = filter(
    lambda i: self._evaluate_tags(self._get_template_tags(i),
                                  settings.ONLY_TAGS,
                                  settings.SKIP_TAGS),
    templates
)
```

**Pull request merged**